contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -152,12 +162,11 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
         }
 
         // Otherwise, this is a token "native" to this chain.
-        require(
-            submittedBridgeTokenCreations[destinationChainID][
-                destinationBridgeAddress
-            ][tokenContractAddress],
-            "Bridge token has not been previously submitted for creation."
-        );
+        if (!submittedBridgeTokenCreations[destinationChainID][
+            destinationBridgeAddress
+        ][tokenContractAddress]) {
```

Comment:
I assume it will return false if tokenContractAddress doesn't in this structure.

Reply:
yes, if the key does not exist in the mapping, it returns default value. The default value of bool is false. https://www.chainlink.education/solidity102/chapter-3